meteor/client/lib/rundownTiming.ts

```diff
@@ -92,8 +101,16 @@ export class RundownTimingCalculator {
 		currentRundown: Rundown | undefined,
 		parts: Part[],
 		partInstancesMap: Map<PartId, PartInstance>,
+		segments: DBSegment[],
 		/** Fallback duration for Parts that have no as-played duration of their own. */
-		defaultDuration?: number
+		defaultDuration?: number,
+		/** The first played-out PartInstance in the current playing segment and
+		 * optionally the first played-out PartInstance in the previously playing segment if the
+		 * previousPartInstance of the current RundownPlaylist is from a different Segment than
+		 * the currentPartInstance.
+		 *
+		 * This is being used for calculating Segment Duration Budget */
+		segmentEntryPartInstances?: PartInstance[] // TODO: Should this actually be optional? Doesn't seem to be used optionally.
```

Comment:
I think this should not be an optional argument, since this method is only being used in a single place and it's always being fed data.